Sources/TuistCore/NodeLoaders/FrameworkNodeLoader.swift

```diff
@@ -49,10 +62,28 @@ public final class FrameworkNodeLoader: FrameworkNodeLoading {
         let linking = try frameworkMetadataProvider.linking(binaryPath: binaryPath)
         let architectures = try frameworkMetadataProvider.architectures(binaryPath: binaryPath)
 
+        let dependencies = try dlybDependenciesPath(forBinaryAt: binaryPath)
+
         return FrameworkNode(path: path,
                              dsymPath: dsymsPath,
                              bcsymbolmapPaths: bcsymbolmapPaths,
                              linking: linking,
-                             architectures: architectures)
+                             architectures: architectures,
+                             dependencies: dependencies)
+    }
+
+    private func dlybDependenciesPath(forBinaryAt binaryPath: AbsolutePath) throws -> [PrecompiledNode.Dependency] {
+        try otoolController
+            .dlybDependenciesPath(forBinaryAt: binaryPath)
+            .toBlocking()
+            .first()?
+            .filter { $0 != binaryPath }
+            .map { $0.removingLastComponent() }
```

Comment:
By looking at this code, I assumed that `dlybDependenciesPath` returns paths to binaries. Am I right? If so, I think we can make the name `dlybDependenciesPath` more explicit. You can perhaps name it `dylibDependenciesBinaryPath`